sdk/core/azure_core/CHANGELOG.md

```diff
@@ -6,8 +6,12 @@
 
 ### Breaking Changes
 
+- Removed `AccessToken::is_expired()`
+
 ### Bugs Fixed
 
+- `BearerTokenCredentialPolicy` returns an error when a proactive token refresh attempt fails
```

Comment:
Especially if this was a customer-filed bug, we reference the bug number such that it renders as `(#{num})`. If not a customer-filed issue, no need.